### PR TITLE
DX: PHP CS Fixer - groom usage

### PR DESCRIPTION
**Pull request by keradus**
DX only, vide commit msgs for reasoning of changes ;)

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -17,26 +17,30 @@ return \PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
+        '@Symfony:risky' => true,
         '@PHP70Migration' => true,
+        '@PHP70Migration:risky' => true,
 
         'concat_space' => ['spacing' => 'one'],
+        'declare_strict_types' => false,
```
Comment (borNfreee) [inline .php_cs.dist]: could you please explain why `false`?

Comment (keradus) [reply .php_cs.dist]: sure I can !
rule is not enabled on master, `@php7:risky` has it by default on, so I turned if off.
it is conflicting with header rule, ending up adding 2 headers (above and below declare statement), for that I didn't wanted to fix everything in single PR ;)

Comment (borNfreee) [reply .php_cs.dist]: this is what @keradus is talking about:

```diff
<?php
/**
 * Copyright © 2017-2018 Maks Rafalko
 *
 * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
 */

declare(strict_types=1);
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
```

Is it a bug of PHP-CS-Fixer?

**Comment by borNfreee** [comment]
Thank you @keradus

File: .php_cs.dist (hunk 1 above)
Comment (keradus) [reply .php_cs.dist]: this is known conflict between those 2 rules, we have raised issue for it for a while, nobody helped solving it so far

**Pull request merged**